charts/cluster-api-provider-openshift-assisted/templates/v1_service_webhook-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
```

Comment:
According to https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/security_and_compliance/configuring-certificates#add-service-serving this should have `service.beta.openshift.io/serving-cert-secret-name=webhook-cert-secret`, not `inject-cabundle`.

Also we should probably make both this service name and the secret name more specific to our application since it's likely the'll end up deployed in the `multicluster-engine` namespace with everything else.

Reply:
:facepalm: sorry this was my first test, I forgot to push the changed chart

however I agree with the generic name, I'll change that and update the PR before merging